package.json

```diff
@@ -3,13 +3,16 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.13.3",
+    "@emotion/styled": "^11.13.0",
```

Comment:
Essa dep serve somente pra fazer styled components? Se for isso acho que não precisa, é mais comum utilizarmos só os [object styles](https://emotion.sh/docs/object-styles)